### ISSUE TITLE
quickstart doesn't ignore files from gitignore even though it states it did

### ISSUE THREAD

**Issue by meetmangukiya** [coala/coala-quickstart#181]
There are no visible ignore keys in the .coafile

**Comment by satwikkansal** [comment]
I think it removes them explicitly from the include paths.

**Comment by satwikkansal** [comment]
Ah, I thinks it's kinda related to https://github.com/coala/coala-quickstart/issues/157

**Comment by ishanSrt** [comment]
This issue is same as https://github.com/coala/coala-quickstart/issues/173

**Comment by satwikkansal** [comment]
@ishanSrt yeah, this needs to be closed. Also, @hemangsk attempted to solve similar issue (#157 ) as part of his PR https://github.com/coala/coala-quickstart/pull/156 which might be helpful to you.
